commands/team_e2e_test.go

```diff
@@ -51,3 +53,40 @@ func (s *MmctlE2ETestSuite) TestRenameTeamCmdF() {
 		s.Equal("Cannot rename team '"+s.th.BasicTeam.Name+"', error : : You do not have the appropriate permissions., ", err.Error())
 	})
 }
+
+func (s *MmctlE2ETestSuite) TestListTeamsCmdF() {
+	s.SetupTestHelper().InitBasic()
+	mockTeamName := "mockteam" + model.NewId()
+	mockTeamDisplayname := "mockteam_display"
+	_, err := s.th.App.CreateTeam(&model.Team{Name: mockTeamName, DisplayName: mockTeamDisplayname, Type: model.TEAM_OPEN, DeleteAt: 1})
+	s.Require().Nil(err)
+
+	s.RunForSystemAdminAndLocal("Should print list of teams", func(c client.Client) {
+		printer.Clean()
+
+		err := listTeamsCmdF(c, &cobra.Command{}, []string{})
+		s.Require().Nil(err)
+		s.Len(printer.GetLines(), 2)
+		team := printer.GetLines()[0].(*model.Team)
+		s.Equal(s.th.BasicTeam.Name, team.Name)
+	})
+
+	s.RunForSystemAdminAndLocal("Should print archived teams", func(c client.Client) {
+		printer.Clean()
+		printer.SetFormat(printer.FormatPlain)
```

Comment:
Would you be open to use the default format instead of plain and do the same type assertion style checks like you're doing in the first test case? It is much more robust to check on the returned structs than on the plaintext messages, so we try to do it if we can